src/util/persistors.js

```diff
@@ -0,0 +1,20 @@
+const genLocalStorageKey = key => `react-transmission.${key}`
+
+export function persistKey (key, val) {
+  window.localStorage.setItem(genLocalStorageKey(key), val);
+}
+
+export function rehydrateKey (key, fallback) {
+  let val = window.localStorage.getItem(genLocalStorageKey(key));
```

Comment:
I think that we can persist the whole object as a JSON object and restore it and don't deal with parse issues